doc/toolbox-create.1.md

```diff
@@ -63,6 +67,25 @@ containers. Read more about the entry-point in `toolbox-init-container(1)`.
 
 ## OPTIONS ##
 
+**--authfile** AUTHFILE
```

Comment:
This option is inherently insecure because it expects people to put their very possibly *secret* credentials in an unencrypted file. Dangerous. :)

Yes, `podman create ...` offers this option, but it doesn't necessarily mean we should offer it. Unless someone shows up with a very valid use-case that can't be achieved in any other way, we shouldn't be offering it.

Reply:
I added the option to match [coreos/toolbox](https://github.com/coreos/toolbox). I also did not realise this particular option can be changed by setting the `REGISTRY_AUTH_FILE` environmental variable. I'll drop the option.